views/movie-details.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="row-detail">
+  {{#with movieId}}
+    <div class="column-picture">
+        <img src="{{this.image}}" alt="{{this.title}}" />
+    </div>
+    <div class="column-description">
+        <a href="/movies">Go back</a>
+        <h1>{{title}}</h1>
+        <h2>{{director}}</h2>
+        <h3>Stars</h3>
+        <ul>
+            <li>{{stars}}</li>
+        </ul>
+        <p>{{description}}</p>
+        <p>{{showtimes}}</p>
```

Comment:
this is also not quite correct. 
you want to have a {{#each}} to loop over the showtimes and display them with a | inbetween each of them...